README.md

```diff
@@ -72,10 +72,12 @@ This function waits as a maximum as the timeout exceeds for a given selector onc
 
 ```js
 await expect(page).toHaveSelector("#foobar")
-// or via not, useful to only wait 1 second instead of for the default timeout by Playwright which is 30 seconds.
-await expect(page).not.toHaveSelector("#foobar", {
-  timeout: 1 * 1000,
-})
+```
+
+When used with `not`, `toHaveSelector` will wait for the element to have the state of `hidden`.
```

Comment:
Thoughts on better way of explaining what is going on here?

Reply:
Maybe just like in playwright documentation?
```
When used with `not`, `toHaveSelector` will wait until the element is not visible or not attached. Note that waiting for hidden does not throw when the element detaches.
```

Reply:
I would also reference to the Playwright documentation.

Reply:
Just updated it.  Have another look when you have a chance.